tests/reference/asr-func_08-c137415.stderr

```diff
@@ -0,0 +1,5 @@
+semantic error: Intent annotation 'In' cannot be used here
+ --> tests/errors/func_08.py:3:17
+  |
+3 | def main0(x: In[In[i32]]):
+  |                 ^^^^^^^ 
```

Comment:
The error message here is shared with the error message for local variables with annotation `In`/ `InOut` / `Out`. See `tests/errors/test_annassign_03.py` in this PR. Therefore the error message is kept slightly general to cover both the cases.